config/migrations/20231108114709-new-minister-data.ttl

```diff
@@ -0,0 +1,124 @@
+        <http://themis.vlaanderen.be/id/mandatee/a3e37299-1321-448f-9d30-d1c921f11d7b>,
+        <http://themis.vlaanderen.be/id/mandatee/ccd0aa72-bad7-48a8-b260-fee95f17a8fd>,
+        <http://themis.vlaanderen.be/id/mandatee/d1290ae5-9a25-4e53-924e-dcfd76cf0886>,
+        <http://themis.vlaanderen.be/id/mandatee/e19a45d5-aaab-4b2e-9ce2-7156219b1999>,
+        <http://themis.vlaanderen.be/id/mandatee/f492c848-0c09-4221-a5aa-3faefd9749cb>,
+        <http://themis.vlaanderen.be/id/mandatee/f6189ea6-eb79-459e-96b9-261eaa960c1e> .
+
+
+<http://themis.vlaanderen.be/id/mandatee/00db38ad-73d7-4d52-a5ba-e8a759d106c0> a ns1:Mandataris ;
+    ns1:isBestuurlijkeAliasVan <http://themis.vlaanderen.be/id/persoon/5fed907ee6670526694a071d> ;
+    ns1:rangorde 9 ;
+    ns1:start "2023-11-08T00:00:00+01:00"^^xsd:dateTime ;
+    ns2:uuid "00db38ad-73d7-4d52-a5ba-e8a759d106c0" ;
+    dcterms:title "Vlaams minister van Brussel, Jeugd en Media" ;
+    org:holds <http://themis.vlaanderen.be/id/mandaat/5fed907ee6670526694a0704> .
+
+<http://themis.vlaanderen.be/id/mandatee/13ba7cb0-d182-4529-9231-8aea9f379ce5> a ns1:Mandataris ;
+    ns1:isBestuurlijkeAliasVan <http://themis.vlaanderen.be/id/persoon/5fed907ee6670526694a0714> ;
+    ns1:rangorde 5 ;
+    ns1:start "2023-11-08T00:00:00+01:00"^^xsd:dateTime ;
+    ns2:uuid "13ba7cb0-d182-4529-9231-8aea9f379ce5" ;
+    dcterms:title "Vlaams minister van Justitie en Handhaving, Omgeving, Energie en Toerisme" ;
+    org:holds <http://themis.vlaanderen.be/id/mandaat/5fed907ee6670526694a0704> .
+
+<http://themis.vlaanderen.be/id/mandatee/1f51946e-b686-43fa-b998-90ede3290001> a ns1:Mandataris ;
+    ns1:isBestuurlijkeAliasVan <http://themis.vlaanderen.be/id/persoon/5fed907ee6670526694a0707> ;
+    ns1:rangorde 1 ;
+    ns1:start "2023-11-08T00:00:00+01:00"^^xsd:dateTime ;
+    ns2:uuid "1f51946e-b686-43fa-b998-90ede3290001" ;
+    dcterms:title "Vlaams minister van Buitenlandse Zaken, Cultuur, Digitalisering en Facilitair Management" ;
+    org:holds <http://themis.vlaanderen.be/id/mandaat/5fed907ee6670526694a0704> .
+
+<http://themis.vlaanderen.be/id/mandatee/25e60e01-f45b-4c55-9e04-1b8f1f1e5417> a ns1:Mandataris ;
+    ns1:isBestuurlijkeAliasVan <http://themis.vlaanderen.be/id/persoon/5fed907de6670526694a061b> ;
+    ns1:rangorde 2 ;
+    ns1:start "2023-11-08T00:00:00+01:00"^^xsd:dateTime ;
+    ns2:uuid "25e60e01-f45b-4c55-9e04-1b8f1f1e5417" ;
+    org:holds <http://themis.vlaanderen.be/id/mandaat/5fed907ee6670526694a0703> .
+
+<http://themis.vlaanderen.be/id/mandatee/671967e1-ad8f-45ad-a1e4-1c20ed5c534e> a ns1:Mandataris ;
+    ns1:isBestuurlijkeAliasVan <http://themis.vlaanderen.be/id/persoon/5fed907ee6670526694a0718> ;
+    ns1:rangorde 6 ;
+    ns1:start "2023-11-08T00:00:00+01:00"^^xsd:dateTime ;
+    ns2:uuid "671967e1-ad8f-45ad-a1e4-1c20ed5c534e" ;
+    dcterms:title "Vlaams minister van Welzijn, Volksgezondheid, Gezin en Armoedebestrijding" ;
```

Comment:
Not a mandataris title that I can find here, mistake or out-of-date page? https://www.vlaanderen.be/vlaamse-regering

Reply:
Mandataris with rangorde 6 should have title "Vlaams minister van Financiën en Begroting, Wonen en Onroerend Erfgoed". This title appears later as rangorde 7, which breaks the composition for later ministers